### PR TITLE
Troubleshooting Deployment

### PR DESCRIPTION
**Pull request by pkarthikr**
@Egis-Moshe Included Deployment Troubleshooting in Configuration

#14

### CODE REVIEW

**Comment by pkarthikr** [comment]
@Egis-Moshe Fixes #4 as well.

**Comment by pkarthikr** [comment]
Added SSO #9

**Comment by pkarthikr** [comment]
Added Advanced Login #21

File: docs/Configuration/Deployment Troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+*  [http://localhost:8080/snapshot](http://localhost:8080/snapshot) will produce a zip file suitable for support diagnosis
+
+*  [http://localhost:8080/web/admin/log.html](http://localhost:8080/web/admin/log.html) to review the current logs. This will also update with new entries.
+
+*  [http://localhost:8080/snapshot/threads](http://localhost:8080/snapshot/threads) to review thread info and CPU usage.
+
+*  [http://localhost:8080/health](http://localhost:8080/health) to check general health.
+
+## Log Levels
```
Comment (Egis-Moshe) [inline docs/Configuration/Deployment Troubleshooting.md]: This is a duplicate of the existing ref/logging.md

File: docs/Configuration/Deployment Troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+## Failure to Start
+
+If PaperTrail fails to start, follow these steps :  
+
+*   Check that __Java is installed__ and no other service is listening on the default port.  
+*   Verify that the __database is up__ and __accepting connections from the configured user/password__.  
+*   Verify that user, service run as user __has appropriate permissions to read and write to all directories__ in the installation directory.  
+*   Review the log files under logs - specifically **error.log, output.log, server.log**.  
+
+## Low Performance
```
Comment (Egis-Moshe) [inline docs/Configuration/Deployment Troubleshooting.md]: Merge/move to Server troubleshooting

File: docs/Configuration/Deployment Troubleshooting.md (hunk 1 above)
Comment (pkarthikr) [inline docs/Configuration/Deployment Troubleshooting.md]: @Egis-Moshe - Could not find this in the existing logging.md.  A couple of links were in CPU Usage.md thought. Should I remove this part entirely then?

Comment (Egis-Moshe) [inline docs/Configuration/Deployment Troubleshooting.md]: http://support.papertrail.co.za/Reference/Logging/

**Comment by pkarthikr** [comment]
@Egis-Moshe Fixed issue with Pull request and merged into Troubleshooting server.

**Pull request merged**